demos/TransferLearning/TransferLearningGui.py

```diff
@@ -0,0 +1,193 @@
+import os
+import random
+import subprocess
```

Comment:
This doesn't seem to be used?

Reply:
os used when pulling test images from directory, random used when gathering random 9 images to display. subprocess removed.